tests/integration/api_v3/http/posStakers.test.js

```diff
@@ -49,6 +51,20 @@ describe('Stakers API', () => {
 			expect(response.data.stakers.length).toBeLessThanOrEqual(10);
 		});
 
+		it('Returns list of stakers when requested for known validator address and search param (exact staker name)', async () => {
+			const response = await api.get(`${endpoint}?address=${refValidator.address}&search=${refStaker.name}`);
+			expect(response).toMap(goodRequestSchema);
+			expect(response.data.stakers.length).toBe(1);
+		});
+
+		it('Returns list of stakers when requested for known validator address and search param (partial staker name)', async () => {
+			const searchParam = refStaker.name ? refStaker.name[0] : '';
```

Comment:
Let us use `refValidator.name` here for both the test cases to ensure that the name is always available. Kindly do the same for all the remaining tests.

Also, kindly consider using `refValidator.name.substring(2)` or something similar than just a single character for the partial string search tests.

Reply:
Refactored to use validator name.